lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1432,3 +1449,12 @@ instance LiftHandler (Request, ServantErr) where
                 , renderHeader $ contentType $ Proxy @JSON
                 ) : headers
             }
+
+instance LiftHandler ErrListStakePools where
+     handler = \case
+         ErrMetricsIsUnsynced p ->
+             apiError err400 NotSynced $ mconcat
+                 [ "I can't list stake pools yet because I need to scan the "
+                 , "blockchain for metrics first. I'm at "
```

Comment:
`for metrics` => `for data` :thinking: ?

Reply:
Maybe `for data` could give the impression that we don't know which stake-pools exist without first scanning the blockchain, which is false.

Reply:
Yes. Although it conflicts with "I can't list stake pools" then. I mean I am neat-picking here ^^"